src/Uno.UI.BindingHelper.Android/Uno/UI/UnoViewGroup.java

```diff
@@ -387,16 +465,15 @@ public boolean dispatchTouchEvent(MotionEvent e)
         boolean isHandlingTouchEvent = _childHandledTouchEvent ||
 				((isBlockingTouchEvent || didPointerExit) && tryHandleTouchEvent(e, _isPointInView, wasPointInView, isCurrentPointer));
 
-        // Log.i(this.toString(), "MotionEvent: " + e.toString());
-        // Log.i(this.toString(), "superDispatchTouchEvent: " + superDispatchTouchEvent);
-        // Log.i(this.toString(), "_childBlockedTouchEvent: " + _childBlockedTouchEvent);
-        // Log.i(this.toString(), "_childHandledTouchEvent: " + _childHandledTouchEvent);
-        // Log.i(this.toString(), "isBlockingTouchEvent: " + isBlockingTouchEvent);
-        // Log.i(this.toString(), "isHandlingTouchEvent: " + isHandlingTouchEvent);
+		//Log.i("", _indent + "superDispatchTouchEvent: " + superDispatchTouchEvent);
+        //Log.i("", _indent + "_childBlockedTouchEvent: " + _childBlockedTouchEvent);
+        //Log.i("", _indent + "_childHandledTouchEvent: " + _childHandledTouchEvent);
+        //Log.i("", _indent + "isBlockingTouchEvent: " + isBlockingTouchEvent);
+        //Log.i("", _indent + "isHandlingTouchEvent: " + isHandlingTouchEvent);
```

Comment:
commented code!

Reply:
Those are only to ease debugging as we are in java and we don't have a valid debugger nor even a real logging system. They were present and helped me a lot, I don't see the point to remove them.